velox/common/memory/MemoryArbitrator.h

```diff
@@ -102,7 +103,7 @@ class MemoryArbitrator {
   /// nullptr, and the memory arbitration function is disabled.
   static std::unique_ptr<MemoryArbitrator> create(const Config& config);
 
-  virtual std::string kind() = 0;
+  virtual const std::string& kind() = 0;
```

Comment:
I think it is fine to leave it as std::string

Reply:
Both OK to me. I can change to std::string.

BTW: It would be great if next time we can avoid change & revert actions around the trivial nits : )... Since it still needs to pick the changed codes carefully. Thanks!